master/buildbot/db/enginestrategy.py

```diff
@@ -109,9 +109,9 @@ def special_case_mysql(self, u, kwargs):
         kwargs['pool_recycle'] = int(u.query.pop('max_idle', 3600))
 
         # default to the InnoDB storage engine
-        storage_engine = u.query.pop('storage_engine', 'InnoDB')
+        storage_engine = u.query.pop('default_storage_engine', 'InnoDB')
```

Comment:
I think we need to make sure that we can setup katana in older version of mysql for backwards compatibility where default_storage_engine may not be defined, I think we could either check the version or allow  storage_engine as parameter in the URL argument